src/Examples/AdversarialExampleGeneration.cs

```diff
@@ -76,20 +79,22 @@ internal static void Main(string[] args)
 
             MNIST.Model model = null;
 
-            var normImage = torchvision.transforms.Normalize(new double[] { 0.1307 }, new double[] { 0.3081 }, device: (Device)device);
+            var normImage = torchvision.transforms.Normalize(new double[] {0.1307}, new double[] {0.3081},
+                device: (Device) device);
 
-            using (var test = new MNISTReader(targetDir, "t10k", _testBatchSize, device: device, transform: normImage)) {
```

Comment:
In the previous iteration, MNISTReader ignored the device, and here it isn't even being passed in. Somewhere, the code needs to exist to place the data tensors (and the model) on the GPU, if that is the desired device. When MNISTReader was only an example, it was obeying the device it was passed.